DATA/production/configurations/asyncReco/async_pass.sh

```diff
@@ -339,32 +339,45 @@ if [[ -n "$ALIEN_JDL_USEGPUS" && $ALIEN_JDL_USEGPUS != 0 ]]; then
   echo "Enabling GPUS"
   export GPUTYPE="HIP"
   export GPUMEMSIZE=$((25 << 30))
-  if [[ "0$ASYNC_PASS_NO_OPTIMIZED_DEFAULTS" != "01" ]]; then
-    if [[ $keep -eq 0 ]]; then
+  if [[ "ALIEN_JDL_USEFULLNUMADOMAIN" == 0 ]]; then
+    if [[ "0$ASYNC_PASS_NO_OPTIMIZED_DEFAULTS" != "01" ]]; then
+      if [[ $keep -eq 0 ]]; then
+	if [[ $ALIEN_JDL_UNOPTIMIZEDGPUSETTINGS != 1 ]]; then
+	  export MULTIPLICITY_PROCESS_tof_matcher=2
```

Comment:
instead of setting this manually, you can now also use `OPTIMIZED_PARALLEL_ASYNC=pp_1gpu`
And below, in the non-GPU settings, instead of setting export OMP_NUM_THREADS=6, we should now use `OPTIMIZED_PARALLEL_ASYNC=pp_8cpu` (for GRID nodes, there is also pp_16cpu for non-1NUMA allocation on the EPNs).

Reply:
Please, check the new version.